tests/llmcompressor/recipe/test_recipe.py

```diff
@@ -52,8 +54,7 @@ def test_recipe_creates_correct_modifier():
     recipe_instance = Recipe.create_instance(yaml_str)
 
     stage_modifiers = recipe_instance.create_modifier()
-    assert len(stage_modifiers) == 1
-    assert len(modifiers := stage_modifiers[0].modifiers) == 1
```

Comment:
I'm surprised we had this before, isn't the `:=` syntax python 3.10+? I would think this would fail the python 3.9 tests.